Server/Controllers/MatchController.cs

```diff
@@ -255,12 +260,24 @@ public async Task<ActionResult> After([FromBody] AfterMatchRequest request)
                     Level = p.Level,
                 })
                 .ToList();
-
             _context.AddRange(newPlayers);
             _context.Matches.Add(match);
+
+            var ratingChanges = request.CustomGame == CustomGame.Dota12v12 ? _ratingService.RecordRankedMatch(match.Players, request.Winner) : null;
+            
             await _context.SaveChangesAsync();
 
-            return Ok();
+            return new AfterMatchResponse()
+            {
+                Players = match.Players
+                    .OrderByDescending(p => p.Player.Rating12v12)
```

Comment:
For that list order shouldn't matter, it gets matched with in-game ids on the client side

Reply:
removed it